src/python/py/models/README.md

```diff
@@ -185,6 +188,24 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options use_8bits_moe=1
 ```
 
+#### Hugging Face Authentication
+
+This scenario is for when you need control over the Hugging Face Authentication, to enable, disable or us a token.
```

Comment:
Can we change this to the following and include the hyperlink?

This scenario is for when you need to disable the Hugging Face authentication or use a different authentication token than the one stored in [`huggingface-cli login`](https://huggingface.co/docs/huggingface_hub/main/en/guides/cli#huggingface-cli-login).